src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -699,7 +725,10 @@ public override void VisitParameter(IParameterSymbol symbol)
                     AddSpace();
                 }
 
-                symbol.Type.Accept(this.NotFirstVisitor);
+                var symbolType = ShouldBeNullableAnnotated(symbol, symbol.Type, FlowAnalysisAnnotations.MaybeNull)
```

Comment:
```suggestion
                var symbolType = ShouldBeNullableAnnotated(symbol, symbol.Type, FlowAnalysisAnnotations.MaybeNullWhenTrue | FlowAnalysisAnnotations.MaybeNullWhenFalse | FlowAnalysisAnnotations.MaybeNull)
```
Parameters are annoted, if `[MaybeNull]`, `[MaybeNullWhen(true)]` or `[MaybeNullWhen(false)]` are present. To make this more obvious I could pass 
`FlowAnalysisAnnotations.MaybeNullWhenTrue | FlowAnalysisAnnotations.MaybeNullWhenFalse | FlowAnalysisAnnotations.MaybeNull`, but `FlowAnalysisAnnotations.MaybeNull` is defined as `MaybeNull = MaybeNullWhenTrue | MaybeNullWhenFalse,`, so it's technically not necessary.

Should I be explicit and add `MaybeNullWhenTrue` and `MaybeNullWhenFalse` to be more coherent?

https://github.com/dotnet/roslyn/blob/f5f54f773cedf07e5c646429be9adad520b401e3/src/Compilers/CSharp/Portable/Symbols/FlowAnalysisAnnotations.cs#L11-L26